resources/tests.mjs

```diff
@@ -125,6 +125,7 @@ Suites.push({
     name: "TodoMVC-JavaScript-ES5",
     url: "resources/todomvc/vanilla-examples/javascript-es5/dist/index.html",
     tags: ["todomvc"],
+    type: "async",
```

Comment:
So in practice all the tests are now async?

Reply:
This can be reverted. For testing I marked all default tests async. 
How do we decide which tests should be async? I know React / Next.js had issues with not capturing all work. Those could be candidates to mark async?

Reply:
I'll leave it unchanged (opting every single workload into async steps), until I get more feedback from other folks.